src/main/java/org/radarbase/appserver/controller/FcmNotificationController.java

```diff
@@ -236,4 +236,20 @@ public ResponseEntity deleteNotificationsForUser(
     this.notificationService.removeNotificationsForUser(projectId, subjectId);
     return ResponseEntity.ok().build();
   }
+
+    @PreAuthorize(
+            AuthConstantsUtil.PERMISSION_ON_SUBJECT_MEASUREMENT_CREATE
+                    + AuthConstantsUtil.ACCESSOR
+                    + AuthConstantsUtil.PROJECT_ID
+                    + ", "
+                    + AuthConstantsUtil.ACCESSOR
+                    + AuthConstantsUtil.SUBJECT_ID
+                    + ")")
+    @DeleteMapping("/" + PathsUtil.MESSAGING_NOTIFICATION_PATH + "/{id}")
```

Comment:
please add project id and subject id in the path as they will be used for authorization.
You will also need to check this in the service if indeed there is a notification with the given ID under the given project and user.